.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,24 @@
+name: WCAG tests
+
+on:
+  push:
```

Comment:
I think just having this run on PRs is sufficient for now! We can adjust if need be later on.